src/config/scrivitoContentBrowser.js

```diff
@@ -103,5 +89,42 @@ Scrivito.configureContentBrowser({
         },
       },
     },
-  },
-});
+  };
+}
+
+function filterOptionForObjClass(objClass) {
```

Comment:
I just realised this is implemented differently from what I thought when reading the call.

The term filter option becomes very confusing here, because is has multiple meanings. This could be a "filter item"?

The term "description" is still used in a misleading way inside of the function body.

* [ ] Can we try to make this clearer?